db/oracle_schema.rb

```diff
@@ -923,555 +924,3 @@
-    t.string "remember_token", limit: 40
-    t.datetime "remember_token_expires_at", precision: 6
-    t.string "activation_code", limit: 40
-    t.datetime "activated_at", precision: 6
-    t.string "state"
-    t.string "role", default: ""
-    t.string "lost_password_token"
-    t.integer "posts_count", precision: 38, default: 0
-    t.integer "account_id", precision: 38
-    t.string "first_name"
-    t.string "last_name"
-    t.string "signup_type"
-    t.string "job_role"
-    t.datetime "last_login_at", precision: 6
-    t.string "last_login_ip"
-    t.string "email_verification_code"
-    t.string "title"
-    t.text "extra_fields"
-    t.integer "tenant_id", precision: 38
-    t.string "cas_identifier"
-    t.datetime "lost_password_token_generated_at", precision: 6
-    t.string "authentication_id"
-    t.string "open_id"
-    t.string "password_digest"
-    t.index ["account_id"], name: "idx_users_account_id"
-    t.index ["email"], name: "index_users_on_email"
-    t.index ["open_id"], name: "index_users_on_open_id", unique: true
-    t.index ["posts_count"], name: "index_site_users_on_posts_count"
-    t.index ["username"], name: "index_users_on_login"
-  end
-
-  create_table "web_hooks", force: :cascade do |t|
-    t.integer "account_id", precision: 38
-    t.string "url"
-    t.boolean "account_created_on", default: false
-    t.boolean "account_updated_on", default: false
-    t.boolean "account_deleted_on", default: false
-    t.boolean "user_created_on", default: false
-    t.boolean "user_updated_on", default: false
-    t.boolean "user_deleted_on", default: false
-    t.boolean "application_created_on", default: false
-    t.boolean "application_updated_on", default: false
-    t.boolean "application_deleted_on", default: false
-    t.datetime "created_at", precision: 6
-    t.datetime "updated_at", precision: 6
-    t.boolean "provider_actions", default: false
-    t.boolean "account_plan_changed_on", default: false
-    t.boolean "application_plan_changed_on", default: false
-    t.boolean "application_user_key_updated_on", default: false
-    t.boolean "application_key_created_on", default: false
-    t.boolean "application_key_deleted_on", default: false
-    t.boolean "active", default: false
-    t.boolean "application_suspended_on", default: false
-    t.integer "tenant_id", precision: 38
-    t.boolean "push_application_content_type", default: true
-    t.boolean "application_key_updated_on", default: false
-  end
-
-  add_foreign_key "api_docs_services", "services"
-  add_foreign_key "payment_details", "accounts", on_delete: :cascade
-  add_foreign_key "policies", "accounts", on_delete: :cascade
-  add_foreign_key "provided_access_tokens", "users"
-  add_foreign_key "proxy_configs", "proxies", on_delete: :cascade
-  add_foreign_key "proxy_configs", "users", on_delete: :nullify
-  add_foreign_key "sso_authorizations", "authentication_providers"
-  add_foreign_key "sso_authorizations", "users"
-end
```

Comment:
what happened with all these lines?

Reply:
Ugh... That's Oracle migration that takes AGES!!! I was actually going to just "cherry-pick" the changes from the other schema files, because my patience is just not enough for this. But accidentally I `git add`ed all of the files.
Thanks, will fix.